### DISCUSSION TITLE
처음에 오고 해야할 일

### DISCUSSION THREAD

**Discussion by shanmdphd** [pipetcpt/trainees#24] [General]
1. Github에 계정을 만듭니다.
2. 그 후 아이디를 shan@catholic.ac.kr 알려주세요.
3. 이 저장소를 Watch / Star 해주세요.
4. Github 저장소 공유 후 다운로드 받거나 git clone 하시면 자료에 접근 가능합니다. 
5. 또는 웹브라우저로 자료를 탐색할 수 있습니다.
6. R과 Rstudio를 설치해 주세요.
7. RStudio 콘솔에 `install.packages('tidyverse')`
8. 다음의 코드를 실행.

```r
library(tidyverse)
ggplot(data=Theoph, aes(x=Time, y=conc, group=Subject)) + 
  geom_line() +
  labs(x="Time (hr)", y="Concentration (mg/L)")
```

**Comment by shanmdphd** [comment]
![image](https://user-images.githubusercontent.com/6946821/211140071-e3de6522-4d2e-48f5-9736-180920e0cfaf.png)

**Comment by Seanjubae** [comment]
4학년 실습 학생

**Reply by Seanjubae** [reply]
https://www.youtube.com/watch?v=w9DfC2BHGPA
